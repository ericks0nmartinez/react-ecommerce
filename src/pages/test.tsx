import { screen, render } from '@testing-library/react'

import Hello from './index'

describe('<Hello />', () => {
  it('should render the heading', () => {
    render(<Hello title="Erickson" />)

    expect(screen.getByRole('heading', { name: /Erickson/i })).toBeInTheDocument
  })
  it('should toMatchSnapshot', () => {
    const { container } = render(<Hello title="Erickson" />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
