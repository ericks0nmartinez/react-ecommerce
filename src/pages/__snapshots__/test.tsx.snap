// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hello /> should toMatchSnapshot 1`] = `
<main>
  <h1>
    Erickson
  </h1>
</main>
`
